README.md

```diff
@@ -6,7 +6,7 @@ A reference implementation of a verifiable credentials platform for digital trad
 
 - VCs exemplify a decentralsied model for high integrity digital data excahnge. There is no central data hub.  Instead there is a global ecosystem of thousands or millions of issuers, verifiers, and holders. A critical success factor is to reduce the cost of entry into the ecosystem so that it is cheap, fast, and simple for new issuers and verifiers to empower their communities with high integrity digital credentials. This project aims to provide free tools and guidance to facilitate uptake.
 
-- This repo also include the renderer (Open-Attestation) to convert VC to nice-looking human readable template
+- This repository also includes the renderer (Open-Attestation) for converting Verifiable Credentials into visually appealing, human-readable, and, if needed, regulatory-compliant templates.
```

Comment:
mixing in doc changes with a chore commit. see other comment, this readme need not have any new content.